src/lib/components/consignments/__stories__/consignments-bottom-aligned.story.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from "react"
```

Comment:
As I'm working with storybook's first, this is the key file to dev against. It contains the different states that a particular component could be. It uses some interesting typescript so I'll explain how that works.